brax/training/agents/ppo/train_test.py

```diff
@@ -131,6 +133,53 @@ def get_offset(rng):
         randomization_fn=rand_fn,
     )
 
+  def testTrainAsymmetricActorCritic(self):
+    """Test PPO with asymmetric actor critic."""
+
+    from brax.envs.fast import Fast
+
+    def _add_privileged_state(state, observation_size):
+      return state.tree_replace({
+        'obs': {
+          'state': state.obs['state'],
+          'privileged_state': jp.zeros(2 * observation_size),
+        }
+      })
+
+    class FastAsymmetric(Fast):
+      def reset(self, rng):
+        reset_state = super().reset(rng)
+        return _add_privileged_state(reset_state, self.observation_size)
+
+      def step(self, state, action):
+        step_state = super().step(state, action)
+        return _add_privileged_state(step_state, self.observation_size)
+
+    network_factory = functools.partial(
+      ppo_networks.make_ppo_networks,
+      policy_state_key='state',
+      value_state_key='privileged_state'
+    )
+
+    _, _, _ = ppo.train(
```

Comment:
can you make an assert on the value/policy param shape, just to make sure the networks are doing what we expect?

Reply:
Done

Reply:
I'm confused, there are no asserts. I figured you can just grab the params returned from `ppo.train` and assert those input shapes

Reply:
Done